saturnbase/install-jupyter.bash

```diff
@@ -20,7 +20,7 @@ jupyter labextension install @ryantam626/jupyterlab_code_formatter@1.3.8
 jupyter labextension install @pyviz/jupyterlab_pyviz
 jupyter labextension install jupyterlab-execute-time
 jupyter labextension install @telamonian/theme-darcula
-jupyter labextension install jupyterlab-python-file
+jupyter labextension install jupyterlab-python-file && conda install numpy
```

Comment:
I intentionally added 4 things we want to prevent, to test that CI will catch all of them

Reply:
looks like that worked! https://github.com/saturncloud/images/pull/163/checks?check_run_id=1750516067

```text
----- checking that image metadata follows Saturn practices ----

saturn
  * directory 'saturn' exists
  * environment.yml exists
  * environment.yml does not have 'prefix:'
  * no pip installs found in Dockerfile
  * no conda installs found in Dockerfile

saturn-gpu
  * directory 'saturn-gpu' exists
  * environment.yml exists
  * environment.yml does not have 'prefix:'
  * no pip installs found in Dockerfile
  * no conda installs found in Dockerfile

saturnbase
  * directory 'saturnbase' exists
  * environment.yml exists
  * environment.yml does not have 'prefix:'
Error: ROR] found 'pip install' in saturnbase/Dockerfile. Update saturnbase/environment.yml instead.
  * no pip installs found in saturnbase/install-jupyter.bash
  * no conda installs found in Dockerfile
Error: ROR] found 'conda install' in saturnbase/install-jupyter.bash. Update saturnbase/environment.yml instead.

saturnbase-gpu
  * directory 'saturnbase-gpu' exists
  * environment.yml exists
  * environment.yml does not have 'prefix:'
  * no pip installs found in Dockerfile
  * no pip installs found in saturnbase-gpu/install-jupyter.bash
Error: ROR] found 'pip install' in saturnbase-gpu/install-miniconda.bash. Update saturnbase-gpu/environment.yml instead.
  * no conda installs found in Dockerfile
  * no conda installs found in saturnbase-gpu/install-jupyter.bash
  * no conda installs found in saturnbase-gpu/install-miniconda.bash
make: *** [Makefile:14: check-metadata] Error 4

blegh
Error: ROR] directory 'blegh' not found

----- errors found while checking images: 4 -------
Error: Process completed with exit code 2.
```

looks like GitHub Actions kind of mangled the output on the ERROR lines though. Let me try making that better.

Reply:
ok this is a lot nicer. 

https://github.com/saturncloud/images/pull/163/checks?check_run_id=1750537075

```text
saturnbase-gpu
  * directory 'saturnbase-gpu' exists
  * environment.yml exists
  * environment.yml does not have 'prefix:'
  * no pip installs found in Dockerfile
  * no pip installs found in saturnbase-gpu/install-jupyter.bash
  * [ISSUE] found 'pip install' in saturnbase-gpu/install-miniconda.bash. Update saturnbase-gpu/environment.yml instead.
  * no conda installs found in Dockerfile
  * no conda installs found in saturnbase-gpu/install-jupyter.bash
  * no conda installs found in saturnbase-gpu/install-miniconda.bash

blegh
  * [ISSUE] directory 'blegh' not found

----- issues found while checking images: 4 -------
make: *** [Makefile:14: check-metadata] Error 4
Error: Process completed with exit code 2.
```

Just changed the text to something that GitHub Actions doesn't have special handling for. It must try to get cute with lines that contain `[INFO]`, `[ERROR]`, and other common log levels